.github/workflows/ci.yml

```diff
@@ -2,7 +2,11 @@ name: CI
 
 on:
   push:
+      branches-ignore:
+          - 'l10n_**'
   pull_request:
+      branches-ignore:
+          - 'l10n_**'
```

Comment:
Would it make sense to still keep it active on pull_request? I guess there's not a ton of applicable tests being run on CI related to translations, but that should at least be one instead of _\*checks notes\*_ 685 times?

Reply:
I'm not sure. I wasn't clear on how that works. Is the PR workflow checking the branch-ignores for the target branch or the branch that is pushed?

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

Reply:
I read that, @Ibochkarev, but I still do not understand what I mentioned above.

Reply:
If I'm right, the statement…

> For a pull_request event, only branches and tags on the base are evaluated.

…means that this would only ignore pull requests if they targeted a base branch that matches the pattern. Let's try it and we can fix it if that is not the case.